### PR TITLE
Fix auto rebase cronjob

### PR DESCRIPTION
**Pull request by fortianyou**

### CODE REVIEW

**Comment by qiuxiafei** [comment]
So what's the solution when conflicts occurs? Simply human take back control and do rebase manually and abandon the PR?

**Comment by fortianyou** [comment]
> So what's the solution when conflicts occurs? Simply human take back control and do rebase manually and abandon the PR?

Yes

File: .github/workflows/cronjobs.yml [hunk 1]
```diff
@@ -6,21 +6,22 @@ on:
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron: '30 23 * * *' # 11:30 pm
-    - cron: '00 00 * * *' # 12:00 pm
```
Comment (Yancey1989) [inline .github/workflows/cronjobs.yml:9]: any fix is onging? maybe It's easier to use `workflow_dispatch ` to test this job manumally.

File: scripts/ci/auto_rebase.sh [hunk 2]
```diff
@@ -11,14 +11,14 @@
...
-commit_msg="[BOT] aicompiler rebase ${date_str}"
+rebase_branch=features/bladedisc_rebase_${date_str}
+commit_msg="[BOT] bladedisc rebase ${date_str}"
 export tf_commit_body=""
 
 function rebase_tf() {
   set -ex
   base_branch=$(git rev-parse HEAD)
-  git config remote.upstream.url >&- || git remote add upstream git@github.com:tensorflow/tensorflow.git
+  git config remote.upstream.url >&- || git remote add upstream https://github.com/tensorflow/tensorflow.git
```
Comment (Yancey1989) [inline scripts/ci/auto_rebase.sh:21]: why not using the tls auth ?

Comment (fortianyou) [reply scripts/ci/auto_rebase.sh:21]: Its permission was denied. See https://github.com/alibaba/BladeDISC/runs/7831036906?check_suite_focus=true

File: .github/workflows/cronjobs.yml (hunk 1 above)
Comment (fortianyou) [reply .github/workflows/cronjobs.yml:9]: The configuration would not work correctly before. The cronjob will start at 11:30 pm and 12:00 pm, but AutoRebase will skip at 12:00 pm. So it does not work as expected and could be better to remove it.

**Pull request merged**